package.json

```diff
@@ -13,14 +13,13 @@
   "author": "Prometheus Authors <prometheus-developers@googlegroups.com>",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@lezer/common": "^0.15.11",
     "@lezer/generator": "^0.15.2",
     "mocha": "^8.1.3",
     "rollup": "^2.27.1",
     "@rollup/plugin-node-resolve": "^9.0.0"
   },
-  "peerDependencies": {
-    "@lezer/common": "^0.15.11"
+  "dependencies": {
```

Comment:
can you keep it as a `peerDependencies` please ?

Reply:
yeqh you are right in a sense where `@lezer/lr` is a dependency of this lib. The fact I prefer to have it as a peerDependency instead is coming from the idea that the user of this lib will have to choose which version of lezer he want to use. (it can be a fresher version). And usually when you are using lezer you are using it with codemirror which brings the lezer dependency on the table.
So here having it as a peer dependency avoid to have a conflict version when you are using this lib and codemirror at the same time.